selfdrive/car/hyundai/hyundaican.py

```diff
@@ -130,7 +130,7 @@ def create_acc_commands(packer, enabled, accel, upper_jerk, idx, lead_visible, s
   commands = []
 
   scc11_values = {
-    "MainMode_ACC": 1,
```

Comment:
I've noticed that this doesn't have a condition check like we see in `scc12` with `ACCMode`. My assumption is that `MainMode_ACC` is the value that's used in long since Main button causes no change with the ACC state.

It's also been a bit unclear to find the exact code that creates the CAN message from buttons within openpilot. The closest I could find is this line that adds ACC commands to can if long is set:
[        can_sends.extend(hyundaican.create_acc_commands(self.packer, CC.enabled, accel, jerk, int(self.frame / 2),](https://github.com/commaai/openpilot/blob/06d681093c7ae9d7b5333000ebc9760fae6e5877/selfdrive/car/hyundai/carcontroller.py#L150)